cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/transformers/JavaActionTransformer.java

```diff
@@ -17,55 +17,45 @@
 import com.google.common.collect.Sets;
 import io.cloudslang.lang.compiler.SlangTextualKeys;
 import io.cloudslang.lang.entities.ScoreLangConstants;
+import org.springframework.stereotype.Component;
+
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
-import org.springframework.stereotype.Component;
 
 @Component
 public class JavaActionTransformer extends AbstractTransformer implements Transformer<Map<String, String>, Map<String, String>> {
-
+    private static DependencyFormatValidator dependencyFormatValidator = new DependencyFormatValidator();
```

Comment:
I think we could define this class as a spring bean (@Component), the autowire it (if you don't go with the `PreCompileValidator` approach)